azure-pipelines/test-pipeline.yml

```diff
@@ -0,0 +1,21 @@
+
+
+stages:
+  - stage:
+
+    jobs:
+    - job: A
+      steps:
+      # assume that MyTask generates an output variable called "MyVar"
+      # (you would learn that from the task's documentation)
+
+      - script: echo '##vso[task.setvariable variable=MyVar;isOutput=true]True'
+        name: ProduceVar  # because we're going to depend on it, we need to name the step
+      - script: echo $(ProduceVar.MyVar)
+    - job: B
+      dependsOn: A
+      variables:
+        # map the output variable from A into this job
+        varFromA: $[ dependencies.A.outputs['ProduceVar.MyVar'] ]
+      steps:
+      - script: echo $(varFromA) # this step uses the mapped-in variable
```

Comment:
this seems like something that doesn't need to be on development

Reply:
I left some deprecated files in by mistake. They should be removed.